requirements.txt

```diff
@@ -1 +1,2 @@
 ansible
+git+https://github.com/evgeni/obsah.git@obsah#egg=obsah
```

Comment:
this is obviously temporary until https://github.com/theforeman/obsah/pull/1 is merged and we published `obsah` to pypi.

Reply:
It has been merged.

Reply:
Sorry, I meant to ask if the plan was to update this with a released Obsah before merging or to use from git for some amount of development time.

Reply:
I think the released version should be fine. Which is what I've updated the file now for.